src/cells.jl

```diff
@@ -37,7 +37,7 @@ CommonMark.block_modifier(c::CellRule) = CommonMark.Rule(100) do parser, node
     if isjuliacode(node) && iscell(node.meta)
         # Load the module for the current cell and evaluate the contents.
         sandbox = getmodule!(c, node)
-        captured = IOCapture.iocapture(throwerrors=false) do
+        captured = IOCapture.iocapture(throwerrors = :interrupt) do
```

Comment:
Could this be the actual exception type instead rather than a symbol?

Reply:
Fredrik had litreally the same suggestion (https://github.com/JuliaDocs/Documenter.jl/pull/1453#discussion_r511652315), so I think it's decided that it goes into IOCapture 0.2 :smile:

Reply:
Would allow for throwing on a `Union` of exception types aside from just a single one I assume?

Reply:
Yes, I was also thinking of taking advantage of type unions, instead of e.g. using a vector of exception types. So the default behaviour is bacially `throwerrors = Any`, and the `false` case.. would be `throwerrors = Union{}`?

While `throwerrors = Union{}` feels a bit complicated (compared to just `false`), it may actually be a good thing. Capturing _all_ errors is rarely what you want, so it might be good if it makes it a bit harder for the user to use that option.

Reply:
Yeah, pretty much that. Currently putting together a pr for it.